Irrgarten_ruby/Monster.rb

```diff
@@ -0,0 +1,47 @@
+#encoding: utf-8
+module Irrgarten
+class Monster
+   @@INITIAL_HEALTH = 5
+   
+   def initialize(name, intelligence, strength, health, row, col)
+       @name = name 
+       @intelligence = intelligence
+       @strength = strength
+       @health = health
+       @row = row
+       @col = col
```

Comment:
```suggestion
    set_pos(0,0)
```
Usar la función que después se implementa.

Reply:
¿Por qué 0,0 ?